tests/gis_tests/gis_migrations/test_operations.py

```diff
@@ -202,6 +202,21 @@ def test_alter_geom_field_dim(self):
         )
         self.assertFalse(Neighborhood.objects.first().geom.hasz)
 
+    def test_oracle_field_arguments(self):
+        """
+        #29792 --- Oracle GIS field argument preservation in migrations.
+        """
+        field = fields.PointField(srid=4067,
+                                  extent=(50199.4814, 6582464.0358, 761274.6247, 7799839.8902),
+                                  tolerance=0.01)
+        name, path, args, kwargs = field.deconstruct()
+        new = fields.PointField(**kwargs)
+
+        self.assertEqual(kwargs['extent'], (50199.4814, 6582464.0358, 761274.6247, 7799839.8902))
+        self.assertEqual(kwargs['tolerance'], 0.01)
+        self.assertEqual(type(new), type(field))
+        self.assertIsNot(new, field)
```

Comment:
Not sure convinced these two last assertions are necessary. You clearly created a new `PointField` instance above so unless a `PointField.__new__` was defined to return a previously defined object this is necessarily going to return a new instance of `PointField`?

Reply:
I think that part was copy-paste from some other test, so I think they're not required.